### PR TITLE
Add IPA demo description, moveit config, and bringup

### PR DESCRIPTION
**Pull request by AdamPettinger**
Addresses #3 and #4. I combined @christianlandgraf's branch and mine as they were somewhat dependent (e.g. the demo description package needed `ur_description` which I added in the repos file (it's in UR ROS 2 driver)

After this you should be able to install and run the launch file as in the README and get to move a UR10e with welding gun around in RViz

### CODE REVIEW

**Comment by AdamPettinger** [comment]
I do not understand why precommit failed the CI here but passed locally when I committed (I have it installed for this repo)

**Comment by christianlandgraf** [comment]
Hey @AdamPettinger , I added an option to the pre-commit config and formatted the description, it seems to work now! I just pushed it to your branch ... Besides that, the Moveit config works for me. I will test it on the real robot  but I guess it could be already merged anyway.
A minor observation: Should we keep only one rviz.config file?

**Comment by AdamPettinger** [comment]
>Besides that, the Moveit config works for me. I will test it on the real robot but I guess it could be already merged anyway.

Beware that I haven't actually tested this on real hardware yet! But I would be interested in knowing how it goes

> A minor observation: Should we keep only one rviz.config file?

Good catch, fixed it

**Comment by christianlandgraf** [comment]
Another thought: Is it cleaner to merge the calibration/ into the config/ directory in `ipa_demo_cell_description`? The DH kinematics of the real robot are currently written in `calibration_ur10e.yaml`, but could replace `default_kinematics.yaml`.

**Comment by AdamPettinger** [comment]
Makes sense to me, done. I renamed from `default_kinematics.yaml` to `calibrated_kinematics` to try and get the point across that the UR10e was using... calibrated numbers

**Review by christianlandgraf — APPROVED** [review]

**Review by sjahr — COMMENTED** [review]
I've tested it in my local setup and everything seems to work. Just the .repos file or the command needs to be updated.

File: README.md [hunk 1]
```diff
@@ -5,3 +5,33 @@ Description: A UR10e welding application demo using online motion planning based
...
+```sh
+export COLCON_WS=~/ws_ur10_welding/ # OR other name
+mkdir -p $COLCON_WS/src
+```
+
+Get the code
+```sh
+cd $COLCON_WS/src
+git clone https://github.com/PickNikRobotics/UR10e_welding_demo.git # Can get specific branch
+vcs import < UR10e_welding_demo/UR10e_welding_demo.repos
```
[the remark this answers is not in the document]
Comment (AdamPettinger) [reply README.md]: Yep, I totally missed that, thanks!

**Comment by christianlandgraf** [comment]
I succesfully tested it on the real robot. I think we should merge it into main to continue work on top of that, if you do not have any remaining concerns?

**Review by henningkayser — APPROVED** [review]

**Pull request merged**